### PR TITLE
Inline Styling for 'Piece of Code' table in Raising and Handling Exceptions

### PR DESCRIPTION
**Pull request by audreyandoy**
Inline Styling for 'Piece of Code' table in Raising and Handling Exceptions #65. Set width to 220px.

### CODE REVIEW

**Review by anselrognlie — APPROVED** [review]

File: fundamentals-recap/math-primer.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The goal of this document is to be a resource. We recommend spending your energy
 
 This document covers the following:
 
-Thinking about numbers:
+Thinking about numbers:
```
Comment (tildeee) [inline fundamentals-recap/math-primer.md:11]:
```suggestion
Thinking about numbers:
```

👀 Kills that irrelevant whitespace in another commit lololol

File: exception-handling/raising-and-handling-exceptions.md [hunk 2]
```diff
@@ -113,14 +113,14 @@ except ExampleError as error_as_a_variable:
     print(f"An exception occurred. Here are the error details: {error_as_a_variable}")
 ```
 
-| Piece of Code             | Notes                                                                                                                                                                                                                                                      |
+| <div style="width:220px;">Piece of Code</div>            | Notes                                                                                                                                                                                                                                                      |
```
Comment (tildeee) [inline exception-handling/raising-and-handling-exceptions.md:116]: It looks great!!!

For what it's worth, as a CSS-opinion-haver...

```suggestion
| <div style="min-width:220px;">Piece of Code</div>            | Notes                                                                                                                                                                                                                                                      |
```

I feel like what's more accurate to our intent is specifying `min-width`. (Also specifying a fixed width with pixels freaks my CSS spidey senses out.)

It produces identical results from what I can see. For your convenience, here's a screenshot of that (from my browser)

<img width="1440" alt="Screen Shot 2021-02-26 at 12 32 15 PM" src="https://user-images.githubusercontent.com/30602862/109351771-ab23a200-782e-11eb-8163-6c63148a91bb.png">

**Review by tildeee — APPROVED** [review]
I'm suggesting `min-width`. I won't fight for it, but I sure will push for it! Otherwise it's an improvement, thanks for making the table readable!

File: exception-handling/raising-and-handling-exceptions.md (hunk 2 above)
Comment (jmaddox19) [reply exception-handling/raising-and-handling-exceptions.md:116]: Yay! Thank you for figuring this out!

**Comment by audreyandoy** [comment]
Merging min-width styling of columns for table in Raising and Handling Exceptions

**Pull request merged**